purchase_request_to_requisition_operating_unit/__openerp__.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# © 2016 Eficent Business and IT Consulting Services S.L.
+# © 2016 Serpent Consulting Services Pvt. Ltd.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "Purchase Request to Call for Bids with Operating Units",
+    "version": "9.0.1.0.0",
```

Comment:
purchase_request_to_requisition_operating_unit [WIP]
purchase_request_to_rfq_operating_unit [WIP]

are in progress please do not review that!! I mentioned that in my earlier comment.